sos/report/plugins/postfix.py

```diff
@@ -52,6 +55,51 @@ def forbidden_ssl_keys_files(self):
         finally:
             return fp
 
+    def forbidden_password_files(self):
+        forbid_attributes = (
+            "lmtp_sasl_password_maps",
+            "smtp_sasl_password_maps",
+            "postscreen_dnsbl_reply_map",
+            "smtp_sasl_auth_cache_name",
+        )
+        fp = []
+        prefix = 'hash:'
+        option_format = re.compile(r"^(.*)=(.*)")
+        try:
+            with open(self.path_join('/etc/postfix/main.cf'), 'r') as cffile:
+                for line in cffile.readlines():
+                    # ignore comment and check option format
+                    line = re.sub('#.*', '', line)
+                    option = option_format.match(line)
+                    if option is None:
+                        continue
+
+                    # sieving
+                    attribute = option.group(1).strip()
+                    if attribute in forbid_attributes:
+                        filepath = option.group(2).strip()
+                        # ignore no filepath
+                        if len(filepath) == 0:
+                            continue
+                        # remove prefix
+                        if filepath.startswith(prefix):
+                            filepath = filepath[len(prefix):]
+                        # ignore non-existent files
+                        if not os.path.exists(filepath):
+                            # warning log for traceability
+                            msg = "'%s' has non-existent path '%s'" % \
+                                    (attribute, filepath)
+                            msg = "[plugin:%s] %s" % (self.name(), msg)
+                            self.soslog.warning(msg)
+                            continue
```

Comment:
This is not needed, imho. The worst what would happen is you feed `add_forbidden_path` with a non-existing file, and any file to be collected would be verified about bit bigger list (https://github.com/sosreport/sos/blob/91cce33a1c9b3cff0795c7e573813d3394cc4107/sos/report/plugins/__init__.py#L1413-L1416).